sensing/autoware_pointcloud_preprocessor/launch/preprocessor.launch.py

```diff
@@ -36,7 +36,7 @@ def launch_setup(context, *args, **kwargs):
     if not is_separate_concatenate_node_and_time_sync_node:
         sync_and_concat_component = ComposableNode(
             package=pkg,
-            plugin="pointcloud_preprocessor::PointCloudConcatenateDataSynchronizerComponent",
```

Comment:
the variable pkg does not have the prefix it seems

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/7983/commits/e19937d087db84bcb6fa0cf9a566eec8031b84ac